src/main/java/com/hubspot/jinjava/lib/tag/ImportTag.java

```diff
@@ -95,6 +95,7 @@ public String interpret(TagNode tagNode, JinjavaInterpreter interpreter) {
         interpreter.render(node);
       } else {
         JinjavaInterpreter child = new JinjavaInterpreter(interpreter);
+        child.getContext().put("importResourcePath", templateFile);
```

Comment:
Since jinjava was directly ported from python we generally use snakecase for context variables.